UTC+01/FR/PAC/FR-PAC-Bouches-du-Rhône/settings.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+#
+# set variables for analysis of network
+#
+
+PREFIX="FR-PAC-Bouches-du-Rhône"
+
+PTNA_TIMEZONE="Europe/Paris"
+
+# avoid downloading same area/data if the data has already been downloaded and is not older than 1 hour (start analysis with: "ptna-networks.sh -fo" to 'f'orce download)
+# OVERPASS_REUSE_ID="FR-PAC-Q3240-train-bus"
+
+OVERPASS_QUERY="http://overpass-api.de/api/interpreter?data=area[wikidata=Q3240][type=boundary];(rel(area)[~'route'~'(train|bus)'];rel(br);rel[type='route'](r);)->.routes;(.routes;<<;rel(r.routes);way(r);node(w);way(r.routes);node(w);node(r.routes););out;"
```

Comment:
I will also filter for all valid route=* instead of limiting this to train and bus.
We can reduce the list after the first analyses.

Reply:
OK let's do it this way. I have some work to do to filter what belongs to these networks and what is not. And the data are not always up to date (some bus lines doesn't exist anymore).